x-pack/test/fleet_api_integration/apis/package_policy/create.ts

```diff
@@ -222,7 +222,7 @@ export default function (providerContext: FtrProviderContext) {
           package: {
             name: 'endpoint',
             title: 'Endpoint',
-            version: '1.4.1',
+            version: '8.4.0',
```

Comment:
ℹ️ The versions of the endpoint plugin this test was written for are super old and predate the addition of an explicit `total_fields` to the `alerts` data stream: https://github.com/elastic/endpoint-package/blob/7d63430d3f65ffe29eec6cac8ceebc6614f1170f/package/endpoint/data_stream/alerts/manifest.yml#L11

This old version is not compatible with this PR as no explicit `total_fields` now means 1000 total fields at most.